bitnami/argo-workflows/values.yaml

```diff
@@ -0,0 +1,1073 @@
+  ## It is also possible to create and manage the certificates outside of this helm chart
+  ## Please see README.md for more information
+  ## e.g:
+  ## secrets:
+  ##   - name: server.local-tls
+  ##     key: |-
+  ##       -----BEGIN RSA PRIVATE KEY-----
+  ##       ...
+  ##       -----END RSA PRIVATE KEY-----
+  ##     certificate: |-
+  ##       -----BEGIN CERTIFICATE-----
+  ##       ...
+  ##       -----END CERTIFICATE-----
+  ##
+  secrets: []
+
+## @section Workflows configuration
+
+workflows:
+  ## Service account configuration for workflows
+  ## @param workflows.serviceAccount.create Whether to create a service account to run workflows
+  ## @param workflows.serviceAccount.name Service account name to run workflows
+  ## @param workflows.serviceAccount.automountServiceAccountToken Automount service account token for the workflows service account
+  ##
+  serviceAccount:
+    create: false
+    name:
+    automountServiceAccountToken: true
+  ## Create RBAC resources to run workflows.
+  ## A Role and Role Bindding are created per namespace in controller.workflowNamespaces
+  ## @param workflows.rbac.create Whether to create RBAC resource to run workflows
+  ##
+  rbac:
+    create: true
+
+## @section PostgreSQL subchart
+
+## Postgresql subchart configuration
+## @param postgresql.enabled Enable PostgreSQL subchart and controller persistence using PostgreSQL
+## @param postgresql.port PostgreSQL port
+## @param postgresql.postgresqlUsername PostgreSQL username
+## @param postgresql.postgresqlDatabase PortgreSQL database name
+##
+postgresql:
+  enabled: false
+  port: 5432
+  postgresqlUsername: postgres
+  postgresqlDatabase: bn_argo_workflows
+
+## @section MySQL subchart
+
+## Mysql subchart configuration
+## @param mysql.enabled Enable MySQL subchart and controller persistence using MySQL
+## @param mysql.service.port MySQL port
+## @param mysql.auth.username MySQL username
+## @param mysql.auth.database MySQL database name
+##
+mysql:
+  enabled: false
+  service:
+    port: 3306
+  auth:
+    username: mysql
+    database: bn_argo_workflows
+
+## @section External Database configuration
+
+## External Database Configuration
+## @param externalDatabase.enabled Enable using externaldatabase and the controller to use persistence with it
+## @param externalDatabase.host External Database server host
+## @param externalDatabase.port External Database server port
+## @param externalDatabase.user External Database username
+## @param externalDatabase.password External Database user password
+## @param externalDatabase.database External Database database name
+## @param externalDatabase.existingSecret The name of an existing secret with database credentials
+## @param externalDatabase.type Either postgresql or mysql
+##
+externalDatabase:
+  enabled: false
+  ## Either "postgres" or  "mysql"
+  ##
+  type: ""
+  host: localhost
+  port: 3306
+  user: bn_wordpress
```

Comment:
Careful, you are using externalDatabase.username in the template

Reply:
ups, a copy paste error